### ISSUE TITLE
[remux] Every 10s: "cat: can't open '/sys/class/power_supply/imx_usb_charger/present': No such file or directory"

### ISSUE THREAD

**Issue by b14d** [rmkit-dev/rmkit#143]
I'm using remux 0.1.9-4 (via Toltec) on rm1 running 2.9.1.236. According to the systemd journal, remux tries to cat `/sys/class/power_supply/imx_usb_charger/present` every 10 seconds, but this sys file does not exist on my device. The closest sys file I have is: `/sys/class/power_supply/bq27441-0/present`.

Output from journalctl -u remux.service:

```
...
Sep 17 03:31:56 reMarkable remux[1540]: cat: can't open '/sys/class/power_supply/imx_usb_charger/present': No such file or directory
Sep 17 03:32:06 reMarkable remux[1553]: cat: can't open '/sys/class/power_supply/imx_usb_charger/present': No such file or directory
Sep 17 03:32:16 reMarkable remux[1566]: cat: can't open '/sys/class/power_supply/imx_usb_charger/present': No such file or directory
Sep 17 03:32:26 reMarkable remux[1601]: cat: can't open '/sys/class/power_supply/imx_usb_charger/present': No such file or directory
Sep 17 03:32:36 reMarkable remux[1620]: cat: can't open '/sys/class/power_supply/imx_usb_charger/present': No such file or directory
...
```

**Comment by raisjn** [comment]
thanks for reporting. this is mostly just log spam and should be quieted down. this path is used to decide whether to go to sleep while USB is connected, it's not a major feature

**Comment by raisjn** [comment]
should be fixed in 8254893999d7334fd14c162d5a15605dcff77ec5 (you can grab latest binary off the build server at https://build.rmkit.dev/master/latest/)

**Comment by b14d** [comment]
Thanks so much. Really appreciate all your efforts. Remux is working well for me.
